security/tailscale/src/opnsense/mvc/app/controllers/OPNsense/Tailscale/forms/settings.xml

```diff
@@ -29,4 +29,11 @@
         <type>checkbox</type>
         <help>Accept subnet routes that other nodes advertise.</help>
     </field>
+    <field>
+        <id>settings.loginTimeout</id>
+        <label>Login timeout</label>
+        <type>text</type>
```

Comment:
Might be worth adding advanced tag, doesn't really need to take up space in UI as rarely see a use case for changing?
```xml
<advanced>true</advanced>
```